docs/further.md

```diff
@@ -373,6 +382,36 @@ rule hello_world:
         "..."
 ```
 
+#### googlebatch_docker_username
+
+For use with the container operating system (COS) if your container needs credentials to access. Both this field and `googlebatch_docker_password` are required.
+
+```console
+rule hello_world:
+	output:
+		"...",
+	resources: 
+		googlebatch_docker_username="monkey"
+	shell:
+        "..."
+```
+
+
+
+#### googlebatch_docker_password
+
+For use with the container operating system (COS) if your container needs credentials to access. Both this field and `googlebatch_docker_username` are required.
+
+```console
+rule hello_world:
+	output:
+		"...",
+	resources: 
+		googlebatch_docker_password="monkey"
+	shell:
+        "..."
```

Comment:
Something like this should never be written in a rule. Also the username like above. Why is that a resource? Is there really a need to specify this per rule? I would just have this as an executor setting.